compiler/plugins/target/AMD-AIE/aie/test/unit_broadcast.mlir

```diff
@@ -24,17 +24,17 @@
 // CHECK:             aie.connect<East : 0, DMA : 0>
 // CHECK:           }
 // CHECK:           %[[SWITCHBOX_2_0:.*]] = aie.switchbox(%[[TILE_2_0]]) {
-// CHECK:             aie.connect<South : 3, North : 0>
 // CHECK:             aie.connect<South : 3, East : 0>
+// CHECK:             aie.connect<South : 3, North : 0>
```

Comment:
`CHECK-DAG` would avoid these reordering issues.

Reply:
![image](https://github.com/user-attachments/assets/c27d4dbf-0d5c-4480-b4ed-17dca7a6273c)

Reply:
`sed -i 's/CHECK/CHECK-DAG/g' unit_broadcast.mlir`

Reply:
Not saying to make the change in this PR though..

Reply:
You can't check-dag everywhere- otherwise you'll allow these to float between switchboxes (that's why I didn't do it last time I wrote these either)

Reply:
Yeah, fair enough. Doing it completely right would indeed be some work